cmake/modules/platform/os/zos.cmake

```diff
@@ -20,12 +20,17 @@
 #############################################################################
 
 list(APPEND OMR_PLATFORM_DEFINITIONS
-	-DJ9ZOS390
-	-DLONGLONG
 	-D_ALL_SOURCE
-	-D_XOPEN_SOURCE_EXTENDED
-	-DIBM_ATOE
+	-D_OPEN_THREADS=2
```

Comment:
It is already defined as part of OpenJ9 [1] and a few other places in OMR port library. I think we should be fine here.

[1] https://github.com/eclipse/openj9/blob/2673d23e5c7525b80254356df1635565f1642ef0/runtime/compiler/build/toolcfg/zos-xlc/common.mk#L75

Reply:
but that applies it to just the jit, not the entirety of omr/openj9, however looking at the docs, it seems it just exposes additional functions rather than changing behavior of any existing functions, so this seems safe enough

Reply:
See also related discussion in [2].

[2] https://github.com/eclipse/omr/pull/3693#discussion_r291634203.